frontend/src/store/modules/website/actions.js

```diff
@@ -1,47 +1,58 @@
 import {SAVE_NEW_WEBSITE, SET_WEBSITE_DATA, UPDATE_WEBSITE} from './types/actions';
-import {UPDATE_CURRENT_WEBSITE, SET_WEBSITE_INFO} from "./types/mutations";
+import {SET_WEBSITE_INFO, UPDATE_CURRENT_WEBSITE} from "./types/mutations";
+import { addWebsite } from '@/api/website';
 
 export default {
+
     [SET_WEBSITE_DATA]: (context, data) => {
         context.commit(SET_WEBSITE_INFO, data);
     },
+
     [SAVE_NEW_WEBSITE]: (context) => {
         const newDataSite = context.state.newWebsite;
 
-        return new Promise((resolve, reject) => {
-            if(!newDataSite.name || !newDataSite.domain) {
-                reject({
-                    errors: {
-                        message: "Fill in all thi fields correctly."
-                    }
-                });
-            }
-            const fakeWebsiteData = {
-                name: 'Website name',
-                domain: 'http://domain.com',
-                single_page: newDataSite.single_page,
-                tracking_number: '123456789-10',
+        if(!newDataSite.name || !newDataSite.domain) {
```

Comment:
Please change this to more specific errors. For instance, if name not set, throw error with key name, if domain not set - error with key domain